README.md

```diff
@@ -28,7 +28,8 @@ See: https://github.com/davidmarkclements/overload-protection/
 
 * options - optional
 * options.readinessURL - string - url where the readiness probe is located
-* options.livenessURLURL - string - url where the livenessURL probe is located
+* options.livenessURL - string - url where the livenessURL probe is located
 * options.readinessCallback - function - function to call when the readiness probe is triggered
 * options.livenessCallback - function - function to call when the liveness probe is triggered
+* options.bypassProtection - boolean - kube-brob will bypass protection and always return 200
```

Comment:
kube-brob should be kube-probe.

Also maybe leave out the "always return 200" since users might think that returns the liveness/readiness callbacks

Reply:
probably want to add that it defaults to true

Reply:
I think you mean that defaults to false cause kube-probe protect the route by default

Reply:
exactly :)